lib/database/models/run.js

```diff
@@ -30,6 +30,20 @@ module.exports = (sequelize) => {
         timeTrgEnd: {
             type: Sequelize.DATE,
         },
+        triggerDuration: {
```

Comment:
Field has to be called `runDuration` and not `triggerDuration`

Reply:
Done, replaced it everywhere